### PR TITLE
Updates for wis2box-training for Jakarta

### PR DESCRIPTION
**Pull request by maaikelimper**
My current WIP has updated:
- Connecting to WIS2 over MQTT: practical-sessions/connecting-to-mqtt.md
- Accessing your student VM: practical-sessions/accessing-your-student-vm.md
- Initializing the wis2box: practical-sessions/wis2box-introduction.md
- Creating wis2box station metadata: practical-sessions/configuring-station-metadata.md

### CODE REVIEW

File: documentation/docs/practical-sessions/connecting-to-mqtt.md [hunk 1]
```diff
@@ -4,38 +4,81 @@ title: Connecting to WIS2 over MQTT
...
+
+## Using MQTT Explorer to connect to the Global Broker
 
-### CMA
+One way to view messages published by this Global Broker is using the MQTT Explorer which can be downloaded from the [MQTT Explorer website](https://mqtt-explorer.com).
+
+Open MQTT Explorer and add a new connection to the Global Broker hosted by CMA using the following details:
 
 - host: gb.wis.cma.cn
 - port: 8883
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/connecting-to-mqtt.md:71]: Do we want to comment about websockets anywhere? Port 1883 and 8883 are not always available.

File: documentation/docs/practical-sessions/connecting-to-mqtt.md [hunk 2]
```diff
@@ -46,48 +89,95 @@ Click on the 'ADVANCED' button and add the following topics to subscribe to:
...
+    (in this case, the Zambia Meteorological Service).
+
+!!! question
+
+    How can we distinguish the data type?
+
+??? note "Click to reveal answer"
 
-    What is the timestamp in UTC for when this data was published?
+    We can distinguish the data type by looking at the ninth of the topic structure.  For example, the following topic:
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/connecting-to-mqtt.md:123]: ninth level?

File: documentation/docs/practical-sessions/connecting-to-mqtt.md (hunk 1 above)
Comment (maaikelimper) [reply documentation/docs/practical-sessions/connecting-to-mqtt.md:71]: Unless we have an example where we can connect over websockets to a Global Broker, 
I do not want to get too technical on day 1 and discuss it here... this session is just meant to illustrate what the WIS2 topics and notifications look like, and to make sure they can use mqtt explorer to see their notifications on the later days when working with their wis2box.

File: documentation/docs/practical-sessions/connecting-to-mqtt.md (hunk 2 above)
Comment (maaikelimper) [reply documentation/docs/practical-sessions/connecting-to-mqtt.md]: Fixed, thanks

File: documentation/docs/practical-sessions/accessing-your-student-vm.md [hunk 3]
```diff
@@ -61,62 +81,47 @@ passwd: password updated successfully
 
 To be able to practice conversion to BUFR, the student VM comes with ecCodes, synop2bufr and csv2bufr pre-installed:
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/accessing-your-student-vm.md:82]: Is this still true or do they now need to use the wis2box management container?

File: documentation/docs/practical-sessions/configuring-station-metadata.md [hunk 4]
```diff
@@ -4,96 +4,132 @@ title: Configuring station metadata
...
+    Please add three or more stations to the wis2box station metadata collection of your wis2box. 
+    
+    Please use stations from your country if possible, especially if you brought your own data.
+    
+    Otherwise, you can use the following WIGOS-station-identifiers for testing purposes:
+
+    - 0-20000-0-91334
+    - 0-20000-0-96323 (note missing station elevation in OSCAR)
+    - 0-20000-0-96749 (note missing station elevation in OSCAR)
+
```
Comment (david-i-berry) [inline documentation/docs/practical-sessions/configuring-station-metadata.md:104]: Add a hint / note that if the elevation is missing there are web services / API that allow you to look up the elevation using open data. One such example is the opentopodata API, e.g. to get the elevvation of the BMKG auditorium we would use https://api.opentopodata.org/v1/aster30m?locations=-6.15558,106.84204

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**